index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio skeleton</title>
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap" rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css?family=Poppins&display=swap" rel="stylesheet" />
+</head>
+
+<body>
+  <main>
+
+    <div class="container"></div>
+      <header>
+        <nav class="navbar">
+          <div class="header-top-art">
+            <img src="./images/image_geometry_1.png" alt="top left art"/>
+          </div>
+       
+          <div class="menu-ham">
+            <img src="/images/icon-menu.svg" alt="ham icon menu"/>
+          </div>
+        </nav>
+      </header>
+
+
+
+      <section class="intro">
+
+        <div>
+          <h1 class="name">Charlie Giorgania</h1>
+          <p class="intro-text">
+            Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my
+            works.
+            If you like what you see and have a project you need coded, don’t hesitate and contact me.
+          </p>
+        </div>
+        <div class="start-collab-box">
+          <div class="start-collab-btn">
+            <a href="#" class="start-content">Start collaboration</a>
+          </div>
+        </div>
+        <div class="art">
+          <img src="./images/rectangle-section-1.svg" alt="deco rectangle"/>
```

Comment:
- [ ] You should add this image as background image to the headline section in CSS instead because it is decoration image and not part of the HTML content. Kindly refer previous link on the Header section.